src/instructlab/model/evaluate.py

```diff
@@ -1,22 +1,113 @@
 # Third Party
 from click_didyoumean import DYMGroup
-from instructlab_eval import eval
+from instructlab.eval import (
+    MMLU_Evaluator,
+    MT_Bench_Evaluator,
+    PR_Bench_Evaluator,
+    PR_MMLU_Evaluator,
+)
 import click
 
-benchmark_names_to_classes = {"mmlu": eval.mmlu, "mt": eval.mt}
+benchmark_names_to_classes = {
+    "mmlu": MMLU_Evaluator,
+    "mt_bench": MT_Bench_Evaluator,
+    "pr_bench": PR_Bench_Evaluator,
+    "pr_mmlu": PR_MMLU_Evaluator,
+}
 
 
 @click.command(cls=DYMGroup)
 @click.option(
-    "--benchmarks",
-    type={},
-    help="benchmarks to run during evaluation",
+    "--benchmark",
+    type=click.Choice(list(benchmark_names_to_classes.keys())),
+    case_sensitive=False,
+    help="Benchmarks to run during evaluation",
+)
+@click.option(
+    "--server-url",
+    type=click.STRING,
+    help="vLLM server endpoint. Needed for running mt_bench or pr_bench.",
+)
+@click.option(
+    "--questions",
+    type=click.STRING,
+    multiple=True,
+    help="Questions to be asked. Needed for running pr_bench.",
+)
+@click.option(
+    "--few-shots",
+    type=click.INT,
+    default=2,
+    show_default=True,
+    help="Number of examples. Needed for running mmlu or pr_mmlu.",
+)
+@click.option(
+    "--batch-size",
+    type=click.INT,
+    default=5,
+    show_default=True,
+    help="Number of GPUs. Needed for running mmlu or pr_mmlu.",
+)
+@click.option(
+    "--tasks",
+    type=click.STRING,
+    multiple=True,
+    help="List of tasks for MMLU to test the model with. Needed for running mmlu.",
```

Comment:
```suggestion
    help="List of tasks for mmlu to test the model with. Needed for running mmlu.",
```

Reply:
MMLU is an acronym, so it should be capitalized